### PR TITLE
Add contact section, and form section

### PR DESCRIPTION
**Pull request by Richie9000**
Add form contact, and submit button, added style to contact section. Added changes to form (field accepts up to 30 chars.
Email field accepts only emails. Message field accepts up to 500 chars.)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -264,6 +264,55 @@ <h3>
...
+          </div>
+          <div class="contact-par">
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need to built or a project that needs coding. I'd
+              love to help with it.
+            </p>
+          </div>
+          <div class="form-container">
+            <form action="http://formspree.io/rr155914@gmail.com" method="post">
```
Comment (Davidosky007) [inline index.html]: I suggest you use the correct action value for form-spree eg (action value should be similar to https://formspree.io/f/{form_id})

File: index.html [hunk 2]
```diff
@@ -264,6 +264,55 @@ <h3>
...
+              <ul class="list-form">
+                <li class="list4">
+                  <input
+                    type="text"
+                    id="name"
+                    name="user_name"
+                    placeholder="Name"
+                  />
+                </li>
```
Comment (Davidosky007) [inline index.html]: * Name input should have `maxlength="30"` attribute. as stated in the project requirements 
* All input fields should have the `required` attribute to activate the validation on submit. as stated in the project requirement

File: index.html [hunk 3]
```diff
@@ -264,6 +264,55 @@ <h3>
...
+                <li class="list4">
+                  <input
+                    type="email"
+                    id="mail"
+                    name="user_email"
+                    placeholder="email"
```
Comment (Davidosky007) [inline index.html]: All input fields should have the `required` attribute to activate the validation on submit. as stated in the project requirement

File: index.html [hunk 4]
```diff
@@ -264,6 +264,55 @@ <h3>
...
+                  />
+                </li>
+              </ul>
+              <textarea
+                id="msg"
+                name="user_message"
+                placeholder="Write your message here"
+              ></textarea>
```
Comment (Davidosky007) [inline index.html]: * All input fields should have the `required` attribute to activate the validation on submit. as stated in the project requirement 
* Message input should have `maxlength="500"` attribute. as  its a requirement for this project

File: index.html [hunk 5]
```diff
@@ -264,6 +264,55 @@ <h3>
...
+              <button type="button" class="btn">Get it Touch</button>
+            </form>
+            <div>
+              <img
+                src="/img/Lightbarrita.png"
+                alt="light-barrita"
+                class="bar-1"
+              />
```
Comment (Davidosky007) [inline index.html]: * This should be a background image

File: index.html [hunk 6]
```diff
@@ -264,6 +264,55 @@ <h3>
...
+                    placeholder="email"
+                  />
+                </li>
+              </ul>
+              <textarea
+                id="msg"
+                name="user_message"
+                placeholder="Write your message here"
+              ></textarea>
+              <button type="button" class="btn">Get it Touch</button>
```
Comment (Davidosky007) [inline index.html]: * When the button is clicked nothing happens please fix this 👍

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Richie9000 @eri8-9, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest your Pull Request have a descriptive title explaining what changes are inside.
* I suggest your  Pull Request have a short summary describing introduced changes in more detail.
* Please check the comments below and apply the necessary changes 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 7]
```diff
@@ -264,6 +264,57 @@ <h3>
...
+              </ul>
+              <textarea
+                id="msg"
+                name="user_message"
+                placeholder="Write your message here"
+                maxlength="500"
+                required
+              ></textarea>
+              <button type="submit" class="btn">Get it Touch</button>
+            </form>
```
Comment (ZawadiSumba66) [inline index.html:311]: - [ ] Make sure the background of your contact form is as of the given design. As requested by the previous reviewer the `img` tag you removed was supposed to be implemented as a background image looking like decorations according to this [figma](https://user-images.githubusercontent.com/68167810/121414822-c181e380-c96f-11eb-8892-215122b3d8a1.png) template. Please fix this

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @Richie9000  & @ eri8-9 ✋  , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_ Please check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Richie9000 , 

I can see that you have made the requested change about the background. :+1: 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**